devel/mps_load_test.sh

```diff
@@ -215,11 +215,11 @@ make -j8 CONF=$compile_mode apache_trace_stress_test_server \
 
 # If a custom .so got specified, install it.
 if [[ -n "$custom_so" ]]; then
-  install -c $custom_so /usr/local/apache2/modules/mod_pagespeed.so
+  install -c $custom_so $HOME/apache2/modules/mod_pagespeed.so
```

Comment:
Is $HOME always right?  For that to go /home/jmarantz/apache2/...

maybe better to have APACHE_DIR which defaults to /usr/local/apache2 if not already defined as an env var.